### PR TITLE
Add Cat surgery

### PR DESCRIPTION
**Pull request by BerSteve**
Added `Cat surgery` docs in main folder and created separate folders for `Dog & Cat` docs under `Tchibo`.

### CODE REVIEW

File: source/products/hansemerkur/cat/webapp.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    <script type="text/javascript" src="https://js.kasko.io"></script>
+    <div id="kasko-widget-container"></div>
+    <script>
+      Kasko.Setup({
+        container: "kasko-widget-container",
+        key: "KEY",
+        touchpoint: "tp_0d65cd1ebdecb251627b360770269",
+        mode: "fullscreen",
+        language: "de"
```
Comment (tomstumshais) [inline source/products/hansemerkur/cat/webapp.rst:17]: If this webapp is using `adnr_number` in snippet like Dog webapp, could you please add it there too?
https://github.com/kasko/docs/blob/master/source/products/hansemerkur/dog-surgery/webapp.rst

File: source/products/hansemerkur/tchibo/cat-surgery/webapp.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    <div id="kasko-widget-container"></div>
+    <script>
+      Kasko.Setup({
+        container: "kasko-widget-container",
+        key: "KEY",
+        touchpoint: "tp_d0f608514295b1a6f2ae07e769b34",
+        mode: "fullscreen",
+        language: "de",
+        config: {
+          adnr_number: "{adnr_number}"
```
Comment (mkasparsone) [inline source/products/hansemerkur/tchibo/cat-surgery/webapp.rst:19]: Hmm does tchibo also have adnr_number? 🤔

Comment (BerSteve) [reply source/products/hansemerkur/tchibo/cat-surgery/webapp.rst:19]: It does not? 🙈

Comment (mkasparsone) [reply source/products/hansemerkur/tchibo/cat-surgery/webapp.rst:19]: Dont know 🤷‍♀️

Comment (mkasparsone) [reply source/products/hansemerkur/tchibo/cat-surgery/webapp.rst:19]: Asked PM, they probably use it. so all good

**Review by mkasparsone — APPROVED** [review]

File: source/products/hansemerkur/tchibo/cat-surgery/webapp.rst (hunk 2 above)
Comment (BerSteve) [reply source/products/hansemerkur/tchibo/cat-surgery/webapp.rst:19]: Cool, thanks!

File: source/products/hansemerkur/tchibo/cat-surgery/webapp.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
+Webapp
+======
+
+Example Kasko JS embed code
+---------------------------
+
+.. code:: html
+
+    <script type="text/javascript" src="https://js.kasko.io"></script>
```
Comment (mkasparsone) [inline source/products/hansemerkur/tchibo/cat-surgery/webapp.rst:9]: shouldnt the script be this one and also the same for non tchibo 🙈? `<script type="text/javascript" src="https://eu1.kaskojs.com/v2"></script>`

Comment (BerSteve) [reply source/products/hansemerkur/tchibo/cat-surgery/webapp.rst:9]: Good catch!

**Review by mkasparsone — APPROVED** [review]

**Review by tomstumshais — APPROVED** [review]

**Pull request merged**